android/app/src/main/java/com/reactnativenavigation/params/SlidingOverlayParams.java

```diff
@@ -17,5 +17,6 @@ public static Position fromString(String string) {
 
     public String screenInstanceId;
     public NavigationParams navigationParams;
+    public Integer autoDismissTimerSec;
```

Comment:
Yes, milliseconds would be superior, though it probably won't matter. X seconds is probably fine grained enough. Also, I also think matching the API that iOS uses is more important.

But I could add `autoDismissTimerMilli` as well as `...Sec`